docs/public/_redirects

```diff
@@ -10,5 +10,7 @@
 
 # Redirections
 
+/ /components/data-grid
```

Comment:
AFAIK this would only impact the Netlify preview, it should work.

```suggestion
/ /components/data-grid/
```

Reply:
Once we have a second component, we could point to the MUI X introduction page

Reply:
Yes the redirection page is the best one as of today but at one point we should have a X home page